WooCommerce/Classes/ViewRelated/Products/Add Product/ManualProductTypeOptions.swift

```diff
@@ -0,0 +1,46 @@
+import SwiftUI
+
+/// View to show the manual product type creation options.
+///
+struct ManualProductTypeOptions: View {
+    private let command: ProductTypeBottomSheetListSelectorCommand
+    private let onOptionSelected: (BottomSheetProductType) -> Void
+
+    init(command: ProductTypeBottomSheetListSelectorCommand,
+         onOptionSelected: @escaping (BottomSheetProductType) -> Void) {
+        self.command = command
+        self.onOptionSelected = onOptionSelected
+    }
+
+    var body: some View {
+        ForEach(command.data) { model in
+            HStack(alignment: .top, spacing: Constants.margin) {
+                Image(uiImage: model.actionSheetImage)
+                    .font(.title3)
+                    .foregroundColor(.secondary)
```

Comment:
The foreground color is not applied to some images. Please ensure the images are rendered as templates for the foreground color to take effect.

Also, we should switch to `foregroundStyle` since `foregroundColor` is deprecating soon.

<img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/79f814a2-bf88-4f17-bb81-b7465762cc99" width=320 />

Reply:
TIL about rendered as template. Updated in https://github.com/woocommerce/woocommerce-ios/pull/11859/commits/f2a78fd450e07e82f76b3e7d1a88ebc236a8ac7e